medusa/providers/torrent/html/extratorrent.py

```diff
@@ -118,44 +112,22 @@ def parse(self, data, mode):
         """
         items = []
 
-        with BS4Parser(data, 'html5lib') as html:
-            torrent_table = html.find('table', class_='tl')
-            torrent_rows = torrent_table('tr') if torrent_table else []
+        with BS4Parser(data, 'html.parser') as xml:
 
-            # Continue only if at least one release is found
-            if len(torrent_rows) < 3 or (torrent_rows == 3 and torrent_rows[2].get_text() == 'No torrents'):
+            elements = xml.find_all('item')
+            if not elements:
                 logger.log('Data returned from provider does not contain any torrents', logger.DEBUG)
                 return items
 
-            # RSS search has one less column
-            decrease = 1
-
-            # Avoid parsing of 'related torrents'
-            if mode != 'RSS':
-                h2s = html.find_all('h2')
-                if len(h2s) > 2 and h2s[1].get_text() == 'Related torrents':
-                    logger.log('Data returned from provider does not contain any torrents', logger.DEBUG)
-                    return items
-
-                # Don't decrease column
-                decrease = 0
-
-            # Skip column headers
-            for result in torrent_rows[2:]:
+            for element in elements:
                 try:
-                    cells = result('td')
-
-                    torrent_info = cells[0].find('a')
-                    if not torrent_info:
+                    title = element.title.get_text()
+                    download_url = element.enclosure.get('url')
```

Comment:
You should use magnets always they are more reliable ,only black hole should use torrent files